lms/envs/common.py

```diff
@@ -1654,13 +1654,15 @@ def _make_locale_paths(settings):
         'source_filenames': [
             'certificates/css/main-ltr.css',
             'css/vendor/font-awesome.css',
+            'edraak-2019-theme/css/certificates/edraak_certificate.css',
```

Comment:
changed to a local URL, the system will load i from theme first as like as other files. Do you see it more convenient @OmarEdraak ?

Reply:
If absolutely necessary then yes, we can do that, otherwise I'd love it of we can avoid as many `edraak-platform` changes as possible. What do you think? I've already issues a draft PR but you don't have to do it: https://github.com/Edraak/edraak-2019-theme/pull/26